hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/services/ITestAbfsInputStreamReadFooter.java

```diff
@@ -54,9 +63,44 @@ public class ITestAbfsInputStreamReadFooter extends ITestAbfsInputStream {
   private static final int TEN = 10;
   private static final int TWENTY = 20;
 
+  private static ExecutorService executorService;
+
+  private final int SIZE_256_KB = 256 * ONE_KB;
+
+  private final Integer[] FILE_SIZES = {
+      SIZE_256_KB,
```

Comment:
The naming format seems not correct as the variables are not static

Reply:
made static.